### DISCUSSION TITLE
Is there an equivalent to `rails db:create`?

### DISCUSSION THREAD

**Discussion by mateusdeap** [loco-rs/loco#102] [Q&A] [answered]
I guess that's pretty much it. I read through the quick tour and searched within the longer guide but couldn't find any mention of a command to create the database via loco rather than starting up a container specifically for that app's db.

**Comment by jondot** [comment, marked as answer]
Good question, it seems that -- how funny it sounds -- this is almost unique to Rails and a few other framework (maybe also Phoenix?).
For some reason unknown to me, outside of the Rails world there is an unwritten rule that databases should be created manually or through the database tools (e.g. `createdb` in postgres).

I actually am positive for providing a `loco db create`, it seems that per database we need to shell out / figure out how to create the database.

If you like you can open a feature request, and we'll take a look. Also if you're into it - we can guide you to submit a PR to implement this youself.

**Reply by mateusdeap** [reply]
Interesting. I guess since I've only ever come into contact with Rails and Phoenix, I kind of expected this to be a thing everywhere hahaha

And yeah, totally can do, both open a feature request and implement it, although my knowledge of Rust is quite rudimentary, so it may take a while.
